doc/release/launcher.rst

```diff
@@ -91,122 +94,309 @@ installation-specific configuration, e.g.), you can often use the ``-v``
 flag to capture the commands that the launcher executes on your behalf
 and customize them for your needs.
 
-
---------------------------------
 Forwarding Environment Variables
---------------------------------
+++++++++++++++++++++++++++++++++
 
 Chapel launchers generally arrange for environment variables to be
 forwarded to worker processes. However, this strategy is not always
 reliable. The remote system may override some environment variables, and
 some launchers might not correctly forward all environment variables.
 
 
----------------------------------
-_real binary suffix for execution
----------------------------------
+.. _using-udp:
 
-In order to support profiling tools that produce new binaries for the
-launcher to execute, the suffix of the real binary executed by the
-launcher may be changed with the ``CHPL_LAUNCHER_SUFFIX`` environment
-variable. If this variable is unset, the suffix defaults to "_real",
-matching the compiler's output.
+Using the Portable UDP Conduit
```

Comment:
A lot of this info about the UDP conduit should be in multilocale.rst rather than here in the launcher doc.  We could talk about the amudprun and slurm-srun launchers here and reference those sections from a UDP conduit section in multilocale.rst, but much of this, for example "I'm seeing warnings from GASNet about using a higher-performance network", has nothing to do with launchers. (Plus, users probably wouldn't come here looking for it.)